apps/judicial-system/backend/src/app/formatters/formatters.ts

```diff
@@ -134,6 +134,28 @@ export function formatCourtResubmittedToCourtSmsNotification(
   })
 }
 
+export function formatReadyForCourtEmailNotificationToProsecutor(
```

Comment:
Consider keeping the naming convention. The name `formatProsecutorReadyForCourtEmailNotification` would be appropriate if you do.

Reply:
Adressed 0f841917846b9d78c85b4332b274739406656f7a